test/test_faiss_and_milvus.py

```diff
@@ -23,12 +24,14 @@
 ]
 
 
+
 @pytest.mark.skipif(sys.platform in ['win32', 'cygwin'], reason="Test with tmp_path not working on windows runner")
-def test_faiss_index_save_and_load(tmp_path):
+def test_faiss_index_save_and_load(tmp_path, sql_url):
     document_store = FAISSDocumentStore(
```

Comment:
Would it simplify things if we use `get_document_store()` here instead? Maybe we can get rid of the `sql_url` fixture then?

Reply:
I'd rather use a fixture than a direct function call. What could be the advantage of using `get_document_store()` here? In general direct use of `get_document_store()` in the tests is something that gave me a lot of pain in this PR, and left me quite puzzled, so I might be missing something crucial...

Reply:
You are right. In general, a fixture would be nicer. 
I think the history of this goes like this: some tests require passing quite special parameters to the document store (here: `progress_bar=False`). Instead of adding this as an "optional parameter" to the `document_store` fixture, we instantiated the document store just within the test. The `get_document_store` is probably something in the middle of the two approaches that avoids duplicate code across a few tests that require the same instantiation without polluting the document_store fixture. 
However, if we find an elegant way to add all those optional params to the document store fixture itself, this would make things probably cleaner.